executor/src/test/java/org/k8loud/executor/kubernetes/BaseTest.java

```diff
@@ -0,0 +1,39 @@
+package org.k8loud.executor.kubernetes;
+
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.kubernetes.client.server.mock.KubernetesServer;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public class BaseTest {
+    protected static final String NAMESPACE = "namespaceValue";
+    protected static final String RESOURCE_NAME = "resourceNameValue";
+    @Mock
+    protected KubernetesClientProvider kubernetesClientProvider;
+    protected KubernetesServer server;
+    protected KubernetesClient client;
+    protected KubernetesService kubernetesService;
+
+    @BeforeEach
+    public void setUp() {
+        server = new KubernetesServer(true, true);
+        server.before();
+        client = server.getClient();
+
+        // Set lenient strictness not to get Unnecessary Stubbing exception
+        Mockito.lenient().when(kubernetesClientProvider.getClient()).thenReturn(client);
```

Comment:
static import. What's more I think it can work without lenient

Reply:
`KubernetesServiceImplTest` fails without it.
Replaced with `@MockitoSettings(strictness = Strictness.LENIENT)`